services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js

```diff
@@ -145,6 +168,35 @@ const Library = ({ libraryData, params, navData }) => {
             </Column>
           </Dashboard>
         </Column>
+        <Column sm={4} md={8} lg={8}>
+          <section>
+            <h2 className={styles['resources-heading']}>Resources</h2>
```

Comment:
This works fine as is, but wondering if we want to use the <H2> component here (everywhere) to get the autolink heading functionality?